openstack/cbr/v3/tags/requests.go

```diff
@@ -0,0 +1,181 @@
+package tags
+
+import (
+	"fmt"
+
+	golangsdk "github.com/opentelekomcloud/gophertelekomcloud"
+)
+
+/*
+ShowVaultProjectTag
+Query the set of all tags of the tenant in the specified Region and instance type
+
+@author Aloento
+@since 0.5.17
+@version 0.1.0
+*/
+func ShowVaultProjectTag(client *golangsdk.ServiceClient) (r TagResult) {
+	_, r.Err = client.Get(showVaultProjectTagURL(client), &r.Body, nil)
+	return
+}
+
+// ----------------------------------------------------------------------------
+
+type ActionType string
+
+const (
+	Filter = "filter"
+	Count  = "count"
+)
+
+type CloudType string
+
+const (
+	Public = "public"
+	Hybrid = "hybrid"
+)
+
+type ObjectType string
+
+const (
+	Server = "server"
+	Disk   = "disk"
+)
+
+type ResourceInstancesRequest struct {
+	// If this parameter is set to true, all resources without tags are queried.
+	WithoutAnyTag bool `json:"without_any_tag,omitempty"`
+	// Returns the full amount of data when there are no filter conditions
+	Tags []Tag `json:"tags,omitempty"`
+	// Backups with any tags in this list will be filtered.
+	TagsAny []Tag `json:"tags_any,omitempty"`
+	// Backups without these tags will be filtered.
+	NotTags []Tag `json:"not_tags,omitempty"`
+	// Backups without any tags in this list will be filtered.
+	NotTagsAny []Tag `json:"not_tags_any,omitempty"`
+	// Only the op_service permission can use this field as a resource instance filter condition
+	SysTags []Tag `json:"sys_tags,omitempty"`
```

Comment:
Please check which tags cannot be set, and remove it from schema

Reply:
We do not support SysTags and have removed them.